kale/prepdata/video_transform.py

```diff
@@ -76,17 +80,19 @@ class ImglistToTensor(torch.nn.Module):
 
     def forward(self, img_list):
         """
-        Converts each PIL image in a list to
-        a torch Tensor and stacks them into
-        a single tensor.
+        Converts each PIL image in a list to a torch Tensor and stacks them into a single tensor.
 
         Args:
             img_list: list of PIL images.
         Returns:
             tensor of size `` NUM_IMAGES x CHANNELS x HEIGHT x WIDTH``
         """
-
-        return torch.stack([transforms.functional.to_tensor(pic) for pic in img_list])
+        img_t = torch.stack([transforms.functional.to_tensor(pic) for pic in img_list])
+        if img_list[0].mode == 'RGB':
+            return img_t
+        elif img_list[0].mode == 'L':
```

Comment:
I assume this is for `flow`. Add some comment to explain will make it easier to read.